src/fix_nh.cpp

```diff
@@ -498,6 +513,29 @@ FixNH::FixNH(LAMMPS *lmp, int narg, char **arg) :
       pre_exchange_flag = 1;
   }
 
+  // checks and overriding defaults for logistic thermostat
+
+  if (logistic_flag) {
+    if (tstat_flag) {
+      if (!mtk_flag)
+        error->all(FLERR,"Cannot use 'mtk no' with 'logistic yes'");
+
+      if (!mtchain_default_flag && mtchain != 1)
+        error->all(FLERR,"Must use 'tchain 1' with 'logistic yes'");
+
+      if (mtchain_default_flag) mtchain = 1;
```

Comment:
This could be made cleaner by reversing the order of testing:

if (mtchain_default_flag) mtchain = 1;
else if (mtchain != 1) error->all(FLERR,"Must use 'tchain 1' with 'logistic yes'");

Same for mpchain.

Reply:
addressed in 12f2809f03447dc47e1e662d32c859aabc8d7482